apps/heft/src/utilities/CoreConfigFiles.ts

```diff
@@ -202,6 +204,21 @@ export class CoreConfigFiles {
     return CoreConfigFiles._typeScriptConfigurationFileLoader;
   }
 
+  public static get sassConfigurationFileLoader(): ConfigurationFile<ISassConfigurationJson> {
+    const schemaPath: string = path.resolve(__dirname, '..', 'schemas', 'sass.schema.json');
+    CoreConfigFiles._sassConfigurationFileLoader = new ConfigurationFile<ISassConfigurationJson>({
+      projectRelativeFilePath: 'config/sass.json',
+      jsonSchemaPath: schemaPath,
+      jsonPathMetadata: {
+        '$.includePaths.*': {
```

Comment:
Seems like the `generatedTsFolder` should also be resolved.

Reply:
Done.